rest/model/pod.go

```diff
@@ -78,10 +63,27 @@ func (p *APICreatePod) ToService() (interface{}, error) {
 			Arch:       arch,
 			EnvVars:    envVars,
 			EnvSecrets: secrets,
+			WorkingDir: utility.FromStringPtr(p.WorkingDir),
 		},
 	}, nil
 }
 
+// fromAPIEnvVars converts a slice of APIPodEnvVars to a map of environment variables and a map of secrets.
+func (p *APICreatePod) fromAPIEnvVars(api []*APIPodEnvVar) (envVars map[string]string, secrets map[string]string) {
```

Comment:
I know this wasn't new for this PR, but...
Would you consider having the function take no arguments and iterating over `p.EnvVars`?

Reply:
Yeah, that makes more sense. I wasn't planning to fix this code really, since it's likely to go away soon after the POC is done, but I changed it.